tests/mypy/test_mypy.py

```diff
@@ -98,6 +98,7 @@ def build(self) -> List[Union[Tuple[str, str], Any]]:
     + [
         ('mypy-plugin.ini', 'custom_constructor.py'),
         ('mypy-plugin.ini', 'generics.py'),
+        ('mypy-plugin.ini', 'root_models.py'),
```

Comment:
How do I generate an output file for this example?

Reply:
You can run `make test-mypy-update-all` — note that it will take a while.

You could probably get it to run faster by commenting out everything but the `root_models.py` file here and dropping the `rm -rf tests/mypy/outputs` from the command (or just reverting all the deletions it will do).